package-lock.json

```diff
@@ -1,703 +1,881 @@
 {
   "name": "typescript-runtime-type-benchmarks",
   "version": "1.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
I think this might be the problem. I still haven't upgraded my NPM and the project still uses the `lockfileVersion: 1`.

Reply:
I'm using node v17 so I think you're right. Let me try re-creating it with an older version.